tests/cupy_tests/functional_tests/test_vectorize.py

```diff
@@ -553,7 +553,7 @@ class TestVectorize(unittest.TestCase):
     @testing.numpy_cupy_allclose(rtol=1e-5)
     def test_vectorize_arithmetic_ops(self, xp, dtype):
         def my_func(x1, x2, x3):
-            y = x1 + x2 * x3 ** x1
+            y = x1 + x2 * x3
```

Comment:
@asi1024 I verified that this test would fail for NumPy on Windows:
```
E               AssertionError: Only numpy raises error
E               
E               Traceback (most recent call last):
E                 File "C:\Windows\TEMP\flexci\run-00142866\work\src\cupy\testing\helper.py", line 47, in _call_func
E                   result = impl(self, *args, **kw)
E                 File "C:\Windows\TEMP\flexci\run-00142866\work\src\tests\cupy_tests\functional_tests\test_vectorize.py", line 564, in test_vectorize_arithmetic_ops
E                   return f(x1, x2, x3)
E                 File "C:\Development\Python\Python37\lib\site-packages\numpy\lib\function_base.py", line 2113, in __call__
E                   return self._vectorize_call(func=func, args=vargs)
E                 File "C:\Development\Python\Python37\lib\site-packages\numpy\lib\function_base.py", line 2200, in _vectorize_call
E                   res = array(outputs, copy=False, subok=True, dtype=otypes[0])
E               OverflowError: Python int too large to convert to C long
```
So here I tweak `my_func` a bit such that it passes on Windows, but I have no idea why the original form would fail. Thoughts?

Reply:
Is it related? https://stackoverflow.com/questions/38314118/overflowerror-python-int-too-large-to-convert-to-c-long-on-windows-but-not-ma

Reply:
This test case is for operator precedence, so I don't want to change this line. How about removing `for_all_dtypes and` fix dtype to int64?

Reply:
Ah, that's probably it, thanks @asi1024! I think somewhere in the test utilities we might need to handle this distinction...

On the tested dtypes, I think it's nice to test a few more dtypes than just int64, so I used `for_dtypes` instead.

Reply:
Thanks!

Reply:
A side question: Do you want to test this? `y = (x1 + x2) * x3` (with parenthesis added) This would test the parenthesis precedence.

Reply:
Yes, but don't need to add the test in this PR. We will add the test in another PR. Thanks!

Reply:
No problem!

Reply:
> On the tested dtypes, I think it's nice to test a few more dtypes than just int64, so I used `for_dtypes` instead.

We still need to do this for `test_vectorize_arithmetic_ops`...